test/features/resource_availability.feature

```diff
@@ -165,44 +167,15 @@ Feature: Re-identification risk governance acknowledgement or Resource visibilit
 
         # Verify the result as non-logged in user.
         Given "Unauthenticated" as the persona
-        And I go to "/dataset/contains-de-identified-data-no"
+        When I go to "/dataset/contains-de-identified-data-no"
         Then I should see "resource created by <User> and is available"
-        Then I should see "resource created by <User> and is available with blank resource_visibility"
-        Then I should not see "resource created by <User> and is NOT available"
+        And I should see "resource created by <User> and is available with blank resource_visibility"
+        And I should not see "resource created by <User> and is NOT available"
+        Then I press the element with xpath "//a[@title='resource created by <User> and is available']"
+        And I should not see "Re-identification risk governance acknowledgement/Resource visibility"
 
         Examples: Users
             | User          |
             | SysAdmin      |
             | TestOrgAdmin  |
             | TestOrgEditor |
-
-
-    Scenario Outline: An editor, admin or sysadmin user views a dataset resource additional info
```

Comment:
why delete dataset from here?

Reply:
nm, the steps are in tests above.

Reply:
please updated tests which were extended 
Scenario Outline: User creates resource for a dataset with Contains de-identified data is NO
to better match what it now does

like

Scenario Outline: User creates resource for a dataset with Contains de-identified data is NO and verifies it being swapped and not seeing settings etc etc